portfolio/templates/portfolio/display.html

```diff
@@ -109,6 +109,7 @@ <h3>Skills</h3>
 				<h3>Projects</h3>
 				<p>Projects that I have worked on. </p>
 				<div class="features">
+					{% if detail.project1_name and detail.project1_description and detail.project1_url %}					
```

Comment:
It's fine but there should be something for the div with class features.If I submit without no projects then I will be able to see an empty block.